cal/cal/views.py

```diff
@@ -10,13 +10,20 @@
 from oauth2client.client import OAuth2WebServerFlow, AccessTokenCredentials
 from oauth2client.django_orm import Storage
 
+from rest_framework.decorators import api_view
+
 from social.apps.django_app.utils import psa
 
 
+@api_view(('GET',))
```

Comment:
You can use `request.GET.get('no_sync')`. We shouldn't mix in django REST framework in the pure Django part of our app.